scripts/install_third_party_libs.py

```diff
@@ -122,9 +122,10 @@ def pip_install(package, version, install_path):
                 'Windows%29')
         raise Exception
 
+    # The following will ensure that Python and Pip versions are compatible.
```

Comment:
Suggest being a tad more specific: "The call to `python -m` is used to ..."

Reply:
done!